### DISCUSSION TITLE
Resouece not found error

### DISCUSSION THREAD

**Discussion by Himanshu728** [tryAGI/LangChain#143] [Q&A]
Unable to find the constructor. The documentation shows (LangChain.Providers.Azure.AzureOpenAIModel) class which is missing in the 0.12.2 version. Currently using this LangChain.Providers.Azure.Gpt4Model class and its function GenerateAsync() it gives "Resource not found" error. The same credentials are working on Azure.AI.OpenAI.OpenAIClient class.

**Comment by HavenDV** [comment]
Hi. Try to use latest preview for now, it will be fixed soon and released as stable version.
